README.md

```diff
@@ -1,15 +1,46 @@
 # magma-access-gateway-operator
 
-The Access Gateway (AGW) provides network services and policy enforcement. In an LTE network, 
-the AGW implements an evolved packet core (EPC), and a combination of an AAA and a PGW. It works 
+The Access Gateway (AGW) provides network services and policy enforcement. In an LTE network,
+the AGW implements an evolved packet core (EPC), and a combination of an AAA and a PGW. It works
 with existing, unmodified commercial radio hardware.
 
-
 ## Usage
 
+Deploying the Magma Access Gateway requires a machine with two network
+interfaces, for the SGi interface (this interface will be used to route traffic
+to the Internet) and the S1 interface (this interface will be used to connect
+to the eNodeB).
+
+Production deployment are highly recommended to be deployed on physical
+hardware.
+
+For testing the deployment, a VM with two DHCP networks attached will do. You
+can use this command to deploy it. The interface names will need to be adjusted
+based on your specific machine.
+
+```bash
+juju deploy magma-access-gateway-operator --config sgi=enp0s1 --config s1=enp0s2
+```
+
+For static network configuration, the easiest method is to use a YAML
+configuration file:
+
+```yaml
+---
```

Comment:
Why do we need `---` at the beginning?

Reply:
It is not strictly required, but this is the YAML start of document. Some YAML linters will complain if it is not present.